wiki/development/coding-guidelines.md

```diff
@@ -61,42 +59,40 @@ class ACE_Settings {
 ### 1.3. Stringtable
 All text that will be displayed to a user shall be defined in a `stingtable.xml` file for multi-language support.
 
-* There will not be any empty stringtable language values.
-* All stringtables will follow the format as specified by Tabler and the [translation guidelines](http://ace3mod.com/wiki/development/how-to-translate-ace3.html) form.
+- There will not be any empty stringtable language values.
+- All stringtables will follow the format as specified by [Tabler](https://github.com/bux578/tabler) and the [translation guidelines](http://ace3mod.com/wiki/development/how-to-translate-ace3.html) form.
 
 
 ## 2. Macro Usage
 
 ### 2.1. Module/PBO specific Macro Usage
 The family of `GVAR` macro's define global variable strings or constants for use within a module. Please use these to make sure we follow naming conventions across all modules and also prevent duplicate/overwriting between variables in different modules. The macro family expands as follows, for the example of the module 'balls':
 
-
 | Macros |  Expands to |
 | -------|---------|
-|`GVAR(face)` | ace_balls_face |
-|`QGVAR(face)` | "ace_balls_face" |
-|`EGVAR(balls,face)` | ace_balls_face |
-|`EGVAR(leg,face)` | ace_leg_face |
-|`QEGVAR(leg,face)` | "ace_leg_face" |
+|`GVAR(face)` | `ace_balls_face` |
+|`QGVAR(face)` | `"ace_balls_face"` |
```

Comment:
It should be, if we make use of all of it. but I don't think we do. We should document what we use and exists within our framework.